src/ai/defaultai.h

```diff
@@ -298,14 +299,16 @@ struct DefaultAI : ComputerPlayer {
 	// the purpose is to print out a warning that the game is pacing too fast
 	int32_t scheduler_delay_counter_;
 
-	WoodPolicy wood_policy_;
+	std::map<Widelands::DescriptionIndex, WoodPolicy> wood_policy_;
```

Comment:
this is now a map to cover for a wood_policy for each ranger building (i.e. one for amazon jungle preservers and a different one for the Rare_tree plantation) to start. stop. build and dismantle each ranger seperatly